src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+
+import { Todo } from '../../types/Todo';
+import { client } from '../../utils/fetchClient';
+
+type Props = {
+  todos: Todo[];
+  setTodos: (a: Todo[] | any) => void;
```

Comment:
Avoid using 'any' type in the function 'setTodos'. It would be better to strictly type it to avoid possible type errors.